deliveries/polkadotjs-ecdsa-hardware-milestone-1.md

```diff
@@ -0,0 +1,14 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://forms.gle/8Wx7nxtq8fKrsuEz8) has been filled out correctly for this milestone.**  
+
+* **PR Link:** [ECDSA for Polkadot JS](https://github.com/w3f/Open-Grants-Program/pull/39)
+* **Milestone Number:** 1
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Trezor API ECDSA signer | [pallet](https://github.com/PlasmNetwork/Plasm/tree/08c4a9211836b929abcbad4ed33ede0f616a6423/frame/custom-signatures), [UI](https://github.com/hoonsubin/apps/pull/2/files) | Trezor support consist of two components: the pallet (that supports any signature schemes to send transactions), and UI that requests Metamask to sign custom data (the `Call` structure). |
+| 2. | Improve documentation | https://docs.plasmnet.io/build/metamask-signatures | Added documentation page that describes provided functionality. |
```

Comment:
Could you update the documentation links in this and the milestone 2 files? Also, in your PR you link to https://docs.astar.network/build/metamask-signatures which doesn't work any longer. Lastly, could you also provide a link to the GitHub repo on which the documentation is hosted so that we can fork it?

Reply:
Sorry for the late response.
@akru can you update the file?